### PR TITLE
*: add a global system variable to control the memory quota for the bindCache

### PR DESCRIPTION
**Pull request by Reminiscent**
…indCache

### What problem does this PR solve?

Issue Number: ref https://github.com/pingcap/tidb/issues/32466

Problem Summary:
add a global system variable to control the memory quota for the bindCache

### What is changed and how it works?
Introduce a global-level system variable

### Check List

Tests 

- [ ] Unit test
- [x] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

**Comment by Reminiscent** [comment]
@morgo PTAL, thanks~

**Review by morgo — APPROVED** [review]

File: sessionctx/variable/sysvar.go [hunk 1]
```diff
@@ -666,6 +666,12 @@ var defaultSysVars = []*SysVar{
 		s.MemQuotaApplyCache = TidbOptInt64(val, DefTiDBMemQuotaApplyCache)
 		return nil
 	}},
+	{Scope: ScopeGlobal, Name: TiDBMemQuotaBindCache, Value: strconv.FormatInt(DefTiDBMemQuotaBindCache, 10), Type: TypeUnsigned, MaxValue: math.MaxInt64, GetGlobal: func(sv *SessionVars) (string, error) {
+		return strconv.FormatInt(MemQuotaBindCache.Load(), 10), nil
```
[the remark this answers is not in the document]
Comment (Reminiscent) [reply sessionctx/variable/sysvar.go:670]: Sure, it can be modified to `MaxInt32` for some protection.

**Review by qw4990 — APPROVED** [review]

**Pull request merged**